src/ui/Forms/Main.cs

```diff
@@ -10551,6 +10551,7 @@ private void SplitSelectedLineBilingual()
             newParagraph.Text = text2;
             SubtitleListview1.Fill(_subtitle, _subtitleAlternate);
             SubtitleListview1.SelectIndexAndEnsureVisible(idx, true);
+            RefreshSelectedParagraph();
```

Comment:
@niksedk That makes more sense.
But I think this line is still needed.

Reply:
@niksedk Look:
![image](https://user-images.githubusercontent.com/20923700/102023025-588c2b00-3d93-11eb-8688-ae61d491c115.png)